week-4/Homework/mandatory/2-writers.js

```diff
@@ -39,6 +39,18 @@ let writers = [
   }
 ];
 
+writers.forEach (e => {
+  if (e.alive === true) { 
+    return console.log(`Hi my name is ${e.firstName} ${e.lastName}. I am ${e.age} years old and my occupation is a ${e.occupation}.`)
+}
+})
+
+
+// writers.forEach(writer => {

```

Comment:
It is good practice to delete the comments after you are done with them and only keep  the final code.